fhir-server/src/main/java/com/ibm/fhir/server/rest/FHIRRestInteractionVisitorReferenceMapping.java

```diff
@@ -143,10 +150,13 @@ public FHIRRestOperationResponse doPatch(int entryIndex, FHIRPersistenceEvent ev
                 addLocalRefMapping(localIdentifier, newResource);
             }
 
-            // TODO support payload offload here
+            // Try offloading storage of the payload. The offloadResponse will be null if not supported
+            String resourcePayloadKey = UUID.randomUUID().toString();
+            int newVersionNumber = Integer.parseInt(newResource.getMeta().getVersionId().getValue());
+            PayloadPersistenceResponse actualOffloadResponse = storePayload(newResource, newResource.getId(), newVersionNumber, resourcePayloadKey);
```

Comment:
Robin and I discussed this one a bit.  He included this here to avoid another pass through the list of bundle entries, but we think it makes sense to split it out into its own phase / interactionVisitor for clarity's sake

Reply:
Robin opened https://github.com/IBM/FHIR/issues/3145 for this

Reply:
ok